src/main/java/io/javalin/core/validation/Validator.kt

```diff
@@ -22,6 +22,11 @@ open class Validator<T>(val value: T?, val messagePrefix: String = "Value") {
 
     fun get(): T = getOrNull() ?: throw BadRequestResponse("$messagePrefix cannot be null or empty")
 
+    fun get(defaultValue: T): T {
+        if (value == null) return defaultValue
+        return rules.find { !it.test.invoke(value) }?.let { defaultValue } ?: value
+    }
```

Comment:
I think this could just be `fun get(defaultValue: T): T = getOrNull() ?: default`

Reply:
Wouldn't it throw exception if value cannot be converted? I tried to avoid that so the function always returns some value.

Reply:
Ah, you might be right about that, I haven't read the code in a while. It has to be possible to reuse some of the code though, what about

`fun get(defaultValue: T): T = if (value == null) defaultValue else get()`

?